clients/js/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@solana-program/token-wrap",
+  "version": "1.0.0",
+  "description": "Javascript helpers for interacting with the Solana Token Wrap program",
+  "author": "Anza Maintainers <maintainers@anza.xyz>",
+  "license": "Apache-2.0",
+  "sideEffects": false,
+  "module": "./dist/src/index.mjs",
+  "main": "./dist/src/index.js",
+  "types": "./dist/types/index.d.ts",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "import": "./dist/src/index.mjs",
+      "require": "./dist/src/index.js"
+    }
+  },
+  "files": [
+    "./dist/src",
+    "./dist/types"
+  ],
+  "scripts": {
+    "build": "tsc && tsup",
+    "build:docs": "typedoc src/index.ts",
+    "lint": "eslint --ext js,ts,tsx src",
+    "lint:fix": "eslint --fix --ext js,ts,tsx src",
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "example:create-mint": "tsx src/example.ts"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solana-program/token-wrap.git"
+  },
+  "bugs": {
+    "url": "https://github.com/solana-program/token-wrap/issues"
+  },
+  "peerDependencies": {
+    "@solana/kit": "^2.1.0"
```

Comment:
You should also add this to `devDependencies` otherwise you won't have it installed locally. Whilst there is no JS tests for this client at the moment, running the example script should also fail. My guess is this is working for you locally right now because the `node_modules` folder still has is from before this change.